tests/app/service/test_statistics.py

```diff
@@ -35,7 +35,9 @@
         StatsRow('email', 'technical-failure', 1),
         StatsRow('email', 'temporary-failure', 1),
         StatsRow('email', 'permanent-failure', 1),
-    ], [4, 0, 4], [0, 0, 0], [0, 0, 0]),
+        StatsRow('email', 'validation-failed', 1),
```

Comment:
Think this would be easier to understand if you changed the type to `letter` here (since emails can’t get into this state):
```suggestion
        StatsRow('letter', 'validation-failed', 1),
```